fem/fe/face_map_utils.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2010-2022, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#ifndef MFEM_FACE_MAP_UTILS_HPP
+#define MFEM_FACE_MAP_UTILS_HPP
+
+#include "../../general/array.hpp"
+#include <utility> // std::pair
+#include <vector>
+
+namespace mfem
+{
```

Comment:
I think it will be better to put these 3 functions inside the `mfem::internal` namespace.

Reply:
Done in da2bd5e42.